Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -87,6 +108,10 @@ open class ChatChannelListVC: _ViewController,
         willDisplay cell: UICollectionViewCell,
         forItemAt indexPath: IndexPath
     ) {
+        if let cellView = (cell as? ChatChannelListCollectionViewCell)?.itemView {
+            cellView.isShowingLoadingState = isShowingLoadingState
+        }
```

Comment:
I'm just thinking if we could have this lines in `cellForRowAt` to handle `content/isShowingLoadingState` in a single place?